tests/unit/workflows/nodejs_npm_esbuild/test_workflow.py

```diff
@@ -307,7 +306,6 @@ def test_workflow_uses_production_npm_version(self, get_workflow_mock):
             download_dependencies=True,
             combine_dependencies=False,
             osutils=self.osutils,
-            experimental_flags=[EXPERIMENTAL_FLAG_ESBUILD],
```

Comment:
nit: this appears to be inconsistent with the other tests that have experimental_flags=[]